src/QMCApp/tests/test_wavefunction_pool.cpp

```diff
@@ -47,12 +47,13 @@ const char *particles = \
       </parameter> \
         <parameter name='LR_dim_cutoff'>15 </parameter> \
 </simulationcell> \
-<particleset name=\"ion\" size=\"1\"> \
-  <group name=\"He\"> \
-    <parameter name=\"charge\">2</parameter> \
+<particleset name=\"ion\" size=\"2\"> \
```

Comment:
The original test pairs a He particle set with a Diamond h5 orbitals. Because of the hybrid rep code, I added a check ensuring atomic centres written in h5 must match xml input based on the tiling matrix.